src/components/application_manager/src/resumption/resumption_data_db.cc

```diff
@@ -130,12 +130,10 @@ bool ResumptionDataDB::Init() {
     return false;
   }
   if (0 == query_checks_resumption.GetInteger(0)) {
-    utils::dbms::SQLQuery query_insert_resumption(db());
-    if (!query_insert_resumption.Prepare(kInsertInitData) ||
```

Comment:
@BSolonenko why `prepare` was removed?

Reply:
@AKalinich-Luxoft  `bool Exec(const std::string& query);` function combines the formatting string in format with the values of the additional arguments to construct an SQL command string and then executes that string on the database referred to by db. 

This is done as in: https://github.com/Ford-Luxoft/sdl_core/blob/d85e7f2e5a844145149a480f8e5d2d8cdcc1a013/src/components/policy/policy_external/src/sql_pt_representation.cc#L440